configuration/parse.go

```diff
@@ -78,9 +84,24 @@ func getCVEExcludesFromFile(config *Configuration, excludeVulnerabilityFilePath
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		line := strings.TrimSpace(scanner.Text())
-		if len(line) > 0 {
-			config.CveList.Cves = append(config.CveList.Cves, line)
+		ogLine := scanner.Text()
+		line := unixComments.ReplaceAllString(ogLine, "")
+		until := untilComment.FindStringSubmatch(line)
+		line = untilComment.ReplaceAllString(line, "")
+		cveOnly := strings.TrimSpace(line)
+
+		if len(cveOnly) > 0 {
```

Comment:
It might be worth breaking out this block into it's own private function. It seems testable as it's own block?

Reply:
Cleaned up in b8a5f0f